packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -15,6 +15,61 @@ import ScrollbarSize from './ScrollbarSize';
 import TabScrollButton from '../TabScrollButton';
 import useEventCallback from '../utils/useEventCallback';
 import tabsClasses, { getTabsUtilityClass } from './tabsClasses';
+import ownerDocument from '../utils/ownerDocument';
+
+const nextItem = (list, item, disableListWrap) => {
+  if (list === item) {
+    return list.firstChild;
+  }
+  if (item && item.nextElementSibling) {
+    return item.nextElementSibling;
+  }
+  return disableListWrap ? null : list.firstChild;
+};
+
+const previousItem = (list, item, disableListWrap) => {
+  if (list === item) {
+    return disableListWrap ? list.firstChild : list.lastChild;
+  }
+  if (item && item.previousElementSibling) {
+    return item.previousElementSibling;
+  }
+  return disableListWrap ? null : list.lastChild;
+};
+
+const moveFocus = (
+  list,
+  currentFocus,
+  disableListWrap,
+  disabledItemsFocusable,
```

Comment:
These are always `false` so let's remove those options for now.

Reply:
I've removed those parameters from the mentioned function as well as the traversal functions.